guardian/media-atom-maker
# Migrate simpler utils and constants to TypeScript

## What does this change?

This migrates some files to TypeScript, to improve our overall type coverage.

It should be a no-op, as no functionality has been changed.

## How to test

Run the app according to the instructions in the readme. Does everything run as expected?
